plugins/st2/auth_token.go

```diff
@@ -0,0 +1,66 @@
+package st2
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func AuthToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.AuthToken,
+		DocsURL:       sdk.URL("https://docs.stackstorm.com/authentication.html"),
+		ManagementURL: sdk.URL("https://docs.stackstorm.com/authentication.html"),
```

Comment:
The management URL should be the address where users can create new tokens/see existing ones.

As an example, for GitHub, that is https://github.com/settings/tokens.

Reply:
If this doesn't exist for the service, you can leave it `nil`

Reply:
> If this doesn't exist for the service, you can leave it `nil`

I don't think there's a way to see these in the UI, but, there's the API endpoint [`POST /auth/v1/tokens`](https://api.stackstorm.com/auth/v1/tokens/#/token_controller.post); not sure if/how to represent this "partial" in `ManagementURL` (I'll push with `nil`).